index.js

```diff
@@ -13,6 +13,9 @@ function cliWidth() {
     if (tty.getWindowSize) {
       return tty.getWindowSize()[1];
     }
+    else if (process.env.CLI_WIDTH) {
+      return parseInt(process.env.CLI_WIDTH, 10);
```

Comment:
@knownasilya `parseInt` returns `NaN` if the variable is not a number. I'll add a test for it and fallback to the default value if so.